src/_210427/boj1072/Main_ja.java

```diff
@@ -0,0 +1,51 @@
+package _210427.boj1072;
+
+import java.util.Scanner;
+
+public class Main_ja {
+	/**
+	 * [실버3] 게임
+	 * 
+	 * 1. 결과 : 틀렸습니다 (5%)
+	 * 2. 시간 복잡도 : O(logN)
+	 * 	  - 게임 횟수(X)를 기준으로 이분탐색
+	 * 	  - 이분 탐색으로 접근한 이유는, 
+	 * 		1_000_000_000(X)에서, 0(Y)번 이겼을 경우가 최악의 경우라고 생각했다.
+	 * 		이때 순차적으로 계산하면 당연히 터질거라고 어림짐작해서 생각했다. 
+	 * 		(이 경우 대략 1억 이상은 넘어야한다고 생각)
+	 * 		계산도 틀렸고, 최악의 경우도 틀렸다.
+	 * 		잘못된 접근인데, 이분탐색이라 생각한 것이다. ㅎㅎ.
+	 * 
+	 * 3. 풀이
+	 * 	  - 게임 횟수(X)를 기준으로 예상 게임 횟수(mid)를 정한다.
+	 * 	  IF. 예상 게임 횟수(mid)로 승률(Z)가 처음이랑 달라졌으면, 범위를 낮춰본다.
+	 * 	  ELSE. 범위를 높여서 다시 확인해본다.
+	 * 
+	 * 4. 후기
+	 * 	  - 자잘하게 신경 써야하는 부분이 많았다. (특히 범위 부분)
+	 * 	  - 문제의 테케는 잘 나오는데 자꾸 틀렸습니다가 나와서 멘탈이 흔들렸다. ㅠㅡㅠ
+	 */
+	public static void main(String[] args) {
+		Scanner in = new Scanner(System.in);
+		int X = in.nextInt();
+		int Y = in.nextInt();
+		int pre = (int)Math.floor((double)Y/X*100);
```

Comment:
```suggestion
		int pre = (int) ((long)Y * 100 / X);
```
double로 변환하고 계산시 double의 오차때문에 맞는 답이 나오지 않음
따라서 최대한 정수 범위 내에서 해결하기 위해 먼저 곱한 뒤 나눠주는 방법을 사용해야 함

그리고 `Math.floor`를 사용하지 않아도 double형을 int에 대입해주면 알아서 소수점 버리고 넣어줌

Reply:
이거땜에 틀렸군요... 슨생님 고마워여 ㅠㅡㅠ
자스 버전도 알려주믄 안대여? 그냥 나눴더니 틀리네 후

Reply:
앗 죄송해영 자스도 똑같네여 ㄳㄳ ㅋㅋㅋㅋ